minio/lxmin
# Fix get backup tags

### Tests:

1. Create and stop vm-broker lxc instance "training0"
<img width="1717" alt="image" src="https://github.com/minio/lxmin/assets/16472240/c7ea4d2e-5c5f-4739-8db6-f40cf4bedd82">

2. Create backup with tags=domain=minio.training using lxmin api
<img width="1728" alt="image" src="https://github.com/minio/lxmin/assets/16472240/527f7af6-024c-4d8e-a45b-0730d608a0c0">

3. Observe backup with tags
<img width="293" alt="image" src="https://github.com/minio/lxmin/assets/16472240/74200c60-9477-4e6b-a0e5-a2f1153d5228">

4. Delete vm-broker lxc instance "training0"
<img width="1728" alt="image" src="https://github.com/minio/lxmin/assets/16472240/a1a24095-5171-4883-91f1-3bfaea01ae35">

5. Restore backup using lxmin api
<img width="1728" alt="image" src="https://github.com/minio/lxmin/assets/16472240/310b755f-fd80-4c2f-864e-f56db34cdd24">

6. Access backup thru software installed locally to validate instance integrity
https://training0.minio.training:9043
<img width="1250" alt="image" src="https://github.com/minio/lxmin/assets/16472240/68db7cd3-6644-4705-89ff-f25eaa95667a">
